components/kyma-environment-broker/internal/process/provisioning/initialisation.go

```diff
@@ -243,3 +260,35 @@ func (s *InitialisationStep) launchPostActions(operation internal.ProvisioningOp
 
 	return s.operationManager.OperationSucceeded(operation, msg)
 }
+
+func (s *InitialisationStep) createTagsForRuntime(operation internal.ProvisioningOperation) ([]*avs.Tag, internal.ProvisioningOperation, time.Duration, error) {
+	instance, err := s.instanceStorage.GetByID(operation.InstanceID)
```

Comment:
you already have an instance in `launchPostActions` method (check arguments) it does not need to fetch again from storage